group-coordinator/src/main/java/org/apache/kafka/coordinator/group/runtime/CoordinatorRuntime.java

```diff
@@ -1242,7 +1304,11 @@ public void scheduleLoadOperation(
                         case FAILED:
                         case INITIAL:
                             context.transitionTo(CoordinatorState.LOADING);
-                            loader.load(tp, context.coordinator).whenComplete((state, exception) -> {
+                            loader.load(tp, context.coordinator).whenComplete((summary, exception) -> {
+                                if (summary != null) {
+                                    log.debug("Finished loading tp={}, LoadSummary={}", tp, summary);
```

Comment:
I am a bit confused here. The log says "Finished loading", but we also schedule an internal operation which is commented as "CompleteLoad"

Reply:
Yeah, this is not the right place to do this. It should rather be done in the event as you said.

Reply:
thanks for the catch!